src/app/modules/admin-core/guards/route.guard.ts

```diff
@@ -13,21 +14,22 @@ export class RouteGuard implements CanActivate {
   rolesArray: any;
   tokendetails: any;
   constructor(private usersService: UsersService, private route: Router,
-    private keycloakService: keyCloakService) {
+    private keycloakService: keyCloakService, private commonServiceService: CommonServiceService) {
+
   }
```

Comment:
Can we make use of this same router guard for all the Pages

Reply:
only we can differentiate the platform_admin and organisation_admin